shopelectro/settings/drone.py

```diff
@@ -5,6 +5,12 @@
 
 DEBUG = True
 
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
```

Comment:
Please, provide a comment for this. Why do we need this backend? How does it relate to header tests?